[2021-11-25 13:01:53,711: INFO: 01_base_model_creation]: 
********************
[2021-11-25 13:01:53,712: INFO: 01_base_model_creation]: >>>>> stage creating base model started <<<<<
[2021-11-25 13:01:53,712: ERROR: 01_base_model_creation]: main() missing 1 required positional argument: 'params_path'
Traceback (most recent call last):
  File "src/01_base_model_creation.py", line 34, in <module>
    main(config_path=parsed_args.config)
TypeError: main() missing 1 required positional argument: 'params_path'
[2021-11-25 13:04:17,130: INFO: 01_base_model_creation]: 
********************
[2021-11-25 13:04:17,130: INFO: 01_base_model_creation]: >>>>> stage creating base model started <<<<<
[2021-11-25 13:04:17,130: ERROR: 01_base_model_creation]: main() missing 1 required positional argument: 'params_path'
Traceback (most recent call last):
  File "src/01_base_model_creation.py", line 66, in <module>
    main(config_path=parsed_args.config)
TypeError: main() missing 1 required positional argument: 'params_path'
[2021-11-25 13:04:46,509: INFO: 01_base_model_creation]: 
********************
[2021-11-25 13:04:46,509: INFO: 01_base_model_creation]: >>>>> stage creating base model started <<<<<
[2021-11-25 13:04:46,510: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2021-11-25 13:04:52,129: INFO: 01_base_model_creation]: >>>>> stage creating base model completed!<<<<<

[2021-11-25 13:18:35,056: INFO: 01_base_model_creation]: 
********************
[2021-11-25 13:18:35,057: INFO: 01_base_model_creation]: >>>>> stage creating base model started <<<<<
[2021-11-25 13:18:35,058: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2021-11-25 13:18:35,414: ERROR: 01_base_model_creation]: module 'tensorflow.keras.layers' has no attribute 'Relu'
Traceback (most recent call last):
  File "src/01_base_model_creation.py", line 66, in <module>
    main(config_path=parsed_args.config)
  File "src/01_base_model_creation.py", line 40, in main
    tf.keras.layers.Relu(), ## alternative way
AttributeError: module 'tensorflow.keras.layers' has no attribute 'Relu'
[2021-11-25 13:19:15,925: INFO: 01_base_model_creation]: 
********************
[2021-11-25 13:19:15,926: INFO: 01_base_model_creation]: >>>>> stage creating base model started <<<<<
[2021-11-25 13:19:15,927: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2021-11-25 13:19:16,354: ERROR: 01_base_model_creation]: module 'tensorflow.keras.layers' has no attribute 'sigmoid'
Traceback (most recent call last):
  File "src/01_base_model_creation.py", line 66, in <module>
    main(config_path=parsed_args.config)
  File "src/01_base_model_creation.py", line 40, in main
    tf.keras.layers.sigmoid(), ## alternative way
AttributeError: module 'tensorflow.keras.layers' has no attribute 'sigmoid'
[2021-11-25 13:19:47,438: INFO: 01_base_model_creation]: 
********************
[2021-11-25 13:19:47,438: INFO: 01_base_model_creation]: >>>>> stage creating base model started <<<<<
[2021-11-25 13:19:47,439: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2021-11-25 13:19:47,847: ERROR: 01_base_model_creation]: module 'tensorflow.keras.layers' has no attribute 'sigmoid'
Traceback (most recent call last):
  File "src/01_base_model_creation.py", line 66, in <module>
    main(config_path=parsed_args.config)
  File "src/01_base_model_creation.py", line 40, in main
    tf.keras.layers.sigmoid(), ## alternative way
AttributeError: module 'tensorflow.keras.layers' has no attribute 'sigmoid'
[2021-11-25 13:19:58,917: INFO: 01_base_model_creation]: 
********************
[2021-11-25 13:19:58,918: INFO: 01_base_model_creation]: >>>>> stage creating base model started <<<<<
[2021-11-25 13:19:58,919: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2021-11-25 13:19:59,301: ERROR: 01_base_model_creation]: module 'tensorflow.keras.layers' has no attribute 'sigmoid'
Traceback (most recent call last):
  File "src/01_base_model_creation.py", line 66, in <module>
    main(config_path=parsed_args.config)
  File "src/01_base_model_creation.py", line 40, in main
    tf.keras.layers.sigmoid(), ## alternative way
AttributeError: module 'tensorflow.keras.layers' has no attribute 'sigmoid'
[2021-11-25 13:20:32,398: INFO: 01_base_model_creation]: 
********************
[2021-11-25 13:20:32,399: INFO: 01_base_model_creation]: >>>>> stage creating base model started <<<<<
[2021-11-25 13:20:32,400: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2021-11-25 13:20:37,525: INFO: 01_base_model_creation]: >>>>> stage creating base model completed!<<<<<

